doc/nrf/gs_installing.rst

```diff
@@ -450,141 +450,23 @@ To set up the toolchain, complete the following steps:
 
 .. rst-class:: numbered-step
 
-Install a build IDE
-*******************
+Install the |VSC| extension
```

Comment:
Hm, it just occurred to me that maybe this should be referring to the entire VSC extension **pack**, instead of just the extension? Otherwise, this may sound like we only want the user to install the nRF Connect extension, and not the others (nRF Konfig, nRF DeviceTree, and nRF Terminal). 

But if the goal in this section is to just instruct users to install only the nRF Connect extension, then this section is fine as is, IMO.

What are your thoughts? @divipillai @greg-fer @Avalei

Reply:
As this section is primarily about the IDE, I have mentioned this as |VSC| extension. @Avalei : Could you confirm whether we should keep this as |VSC|, |VSC| extension or |VSC| extension pack.

Reply:
Let's drop the usage of both "extension" or "extension pack" and simply say "Install nRF Connect for VS Code". This section will direct the user to download the extension pack anyway in the VSC docs, so I don't think it's necessary to state it multiple times.